docs/_docs/modules/analytics.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Module: Analytics"
+---
+
+{% include under-construction.html %}
```

Comment:
Should this be here? Is it still under construction?

Reply:
Good catch :)